js/app.js

```diff
@@ -82,29 +62,37 @@ const main = () => {
     });
   });
 
-  app.config([
-    '$translateProvider',
-    ($translateProvider) => {
-      // Register a loader for the static files
-      $translateProvider.useStaticFilesLoader({
-        prefix: './i18n/',
-        suffix: '.json',
-      });
-      $translateProvider.use('zh_CN');
-      // Tell the module what language to use by default
-      $translateProvider.preferredLanguage('zh_CN');
-      $translateProvider.useSanitizeValueStrategy('escape');
-    },
-  ]);
-
   app.run([
-    '$q',
-    '$translate',
-    ($q, $translate) => {
+    '$q', ($q) => {
       axios.Axios.prototype.request_original = axios.Axios.prototype.request;
       axios.Axios.prototype.request = function new_req(config) {
         return $q.when(this.request_original(config));
       };
+      window.notyf = new Notyf({
+        duration: 5000,
+        ripple: false,
+        position: { x: 'center', y: 'top' },
+        types: [
+          {
+            type: 'warning',
+            background: 'darkorange',
+            icon: false,
+          },
+          {
+            type: 'info',
+            background: 'deepskyblue',
+            icon: false,
+          },
+        ],
+      });
+      window.notyf.warning = (msg) => window.notyf.open({
```

Comment:
bug report: notice will show [Object object] instead of text content. Maybe parameter format is not correct.